src/components/SiteFooter.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import '../styles/components/SiteFooter.css';
+
+export default function SiteFooter(){
```

Comment:
Lint maybe? XD

Reply:
It's formatted  @lucasmauro, accordingly with the most used linter for react (eslint) 😢 
Ps.: I've added a lint check on Travis CI as part of the CI workflow, it checks the lint and then run the tests).